app/jobs/backfill_marked_as_flagged.rb

```diff
@@ -0,0 +1,23 @@
+class BackfillMarkedAsFlagged < ApplicationJob
+  def perform(start: 0, finish: nil)
```

Comment:
Is the intent that you'll run this job manually in `rails c`? If so, this makes sense. Wasn't entirely sure what the usage pattern was going to be, so figured I'd ask.

Reply:
If that's how you'll do it, then I guess you'll let it run and watch the logs for the output at the end? That makes sense to me.